packages/compass-e2e-tests/tests/auto-update.test.ts

```diff
@@ -0,0 +1,78 @@
+import { expect } from 'chai';
+import {
+  init,
+  cleanup,
+  Selectors,
+  screenshotPathName,
+} from '../helpers/compass';
+
+function wait(ms: number) {
+  return new Promise((resolve) => {
+    setTimeout(resolve, ms);
+  });
+}
+
+describe('Auto-update', function () {
+  it('auto-update from', async function () {
+    if (process.env.TEST_NAME !== 'auto-update-from') {
+      // we don't want this test to execute along with all the others under
+      // normal circumstances because it is destructive - it overwrites Compass
+      // itself
+      this.skip();
+    }
+
+    // run the app and wait for it to auto-update
+    console.log('starting compass the first time');
```

Comment:
I have another branch open where I'm switching from console.log to debug. I don't want to conflict with that now, so will fix all these together in a future PR.